### PR TITLE
feat(backdrop): backdrop component

### PR DESCRIPTION
**Pull request by brunovgois**
image

### CODE REVIEW

File: packages/backdrop/src/lib/backdrop.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+export function Backdrop(props: BackdropProps) {
+  const { children, ...other } = props;
+
+  return (
+    <MUIBackdrop
+      {...other}
+      sx={{
+        zIndex: (theme) => theme.zIndex.drawer + 1,
```
Comment (mgonc) [inline packages/backdrop/src/lib/backdrop.tsx:18]: why are we changing the mui zIndex?

File: packages/backdrop/src/lib/backdrop.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        backgroundColor: 'rgba(128, 128, 128, 0.5) ',
```
Comment (mgonc) [inline packages/backdrop/src/lib/backdrop.tsx:19]: we have a function addTransparency at @novatics/styles

**Review by mgonc — CHANGES_REQUESTED** [review]
Since we are only changing css properties, we don't need to create a component. Let's just use the Theme approach

File: packages/backdrop/src/lib/backdrop.tsx (hunk 2 above)
Comment (brunovgois) [reply packages/backdrop/src/lib/backdrop.tsx]: Fixed on new commit

**Review by mgonc — APPROVED** [review]

**Pull request merged**